utils/charts/lineChart.js

```diff
@@ -0,0 +1,56 @@
+const generateLineChartForData = (months, trueCosts, wouldBeCosts) => {
+  return {
+    title: {
+      text: ''
+    },
+    yAxis: {
+      title: {
+        text: ''
+      },
+      labels: {
+        format: '${value:,.0f}'
+      }
+    },
+    xAxis: {
+      categories: months
+    },
+    series: [
+      {
+        name: "What You've Paid",
+        data: trueCosts,
+        color: '#cd6795'
+      },
+      {
+        name: '"Would-be" costs',
+        data: wouldBeCosts,
+        color: '#747474'
```

Comment:
maybe declare the colors as constants at the top of the file, or together in a separate colors constants file?